pkg/console/operator/sync_v400.go

```diff
@@ -57,12 +57,23 @@ func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console,
 	//     at the same resource.
 	//     - RouteSyncController is responsible for updates
 	//     - ConsoleOperatorController (future ConsoleDeploymentController) is responsible for reads only.
-	status.HandleProgressingOrDegraded(updatedOperatorConfig, "SyncLoopRefresh", "InProgress", rtErr)
-	if rtErr != nil {
-		return rtErr
+	status.HandleProgressingOrDegraded(updatedOperatorConfig, "SyncLoopRefresh", "InProgress", defaultRouteErr)
+	if defaultRouteErr != nil {
+		return defaultRouteErr
+	}
+
+	// Need to make difference of the default and active route due to the custom console hostname.
+	activeRoute := defaultRoute
+	if routesub.IsCustomRouteSet(updatedOperatorConfig) {
+		customRoute, customRouteErr := co.routeClient.Routes(api.OpenShiftConsoleNamespace).Get(api.OpenshiftConsoleCustomRouteName, metav1.GetOptions{})
+		status.HandleProgressingOrDegraded(updatedOperatorConfig, "CustomRoutePropagation", "FailedGetCustomRoute", customRouteErr)
+		if customRouteErr != nil {
+			return customRouteErr
+		}
+		activeRoute = customRoute
```

Comment:
```go
routeName := api.OpenShiftConsoleName
if routesub.IsCustomRouteSet(updatedOperatorConfig) {
    routeName := api.OpenShiftConsoleCustomRouteName
}

doMagicHere()
```
also, please don't use `rt` as a variable name for `route`, `rt` usually stands for `roundTripper`, this could be confusing a random passerby.